application/helpers/admin/import_helper.php

```diff
@@ -1419,6 +1419,17 @@ function XMLImportSurvey($sFullFilePath, $sXMLdata = null, $sNewSurveyName = nul
         $surveyLanguageSetting = new SurveyLanguageSetting();
         $surveyLanguageSetting->setAttributes($insertdata, false);
         try {
+            // Clear alias if it was already in use
+            $surveyLanguageSetting->checkAliasUniqueness();
```

Comment:
checkAliasUniqueness() - this sounds like it should return a boolean or something. But old function, or?

Reply:
That's a validation rule for the model. Not sure if it can handle return values, so it can be called outside a model validation.